### ISSUE TITLE
Fix Ships selling for too much

### ISSUE THREAD

**Issue by captaincobbs** [captaincobbs/ESG-Mod#370]
Lowered ship sell price by 50%

**Comment by Doruk-Coskun** [comment]
(defense+offense value) / Ship CP * (1 + (Ship Level - 1) / 20)

**Comment by captaincobbs** [comment]
waiting for #377

**Comment by wakystuf** [comment]
I just found in the code a place where it looks like this may have been implemented? I'm tagging as Needs Documentation just in case, so that I don't lose track

**Comment by Doruk-Coskun** [comment]
@wakystuf  It is not changed yet. We will hopefully fix it in 1.3.2 with the economy

**Comment by wakystuf** [comment]
To clarify, the real version is not implemented, but the 50% reduction band-aid is implemented, right?

**Comment by Doruk-Coskun** [comment]
huh I actually dont know

**Comment by wakystuf** [comment]
OK. I think I saw it yesterday when I was reading through all of the 1.3.1 code, but I don't remember where

**Comment by Arpharnator** [comment]
Pretty sure it's implemented cuz my ships sell for 15 pennies

**Comment by wakystuf** [comment]
OK. I'm going to create a separate Issue for the band-aid, which I'll label as Documented, so that I can remove Needs Documentation from this one (since that's how I track what I still need to work on)

**Comment by Arpharnator** [comment]
works, ships sell for absolutely nothing

**Comment by wakystuf** [comment]
Re-implemented this with a partial version of the real solution (vs the previous band-aid of "vanilla / 2")

Ship costs calculated from:
* Ind cost of hull (not modules) * 40% +
* Strat cost of hull (not modules or expansion slots) * strat pseudoprices (calculated from Empire stock, since market variables apparently aren't accessible)

**Comment by wakystuf** [comment]
Potentially in a later update could also account for costs of modules, but only if @captaincobbs  can feed me the variables again :)

**Comment by captaincobbs** [comment]
Waky's ever-present ravenousness for variables is terrifying

**Comment by captaincobbs** [comment]
Halfway finished, dependent on #385 to completely finish
